openff/interchange/tests/components/test_interchange.py

```diff
@@ -83,6 +73,61 @@ def test_basic_combination(self):
         get_openmm_energies(combined)
 
 
+class TestUnimplementedSMIRNOFFCases(BaseTest):
+    def test_bogus_smirnoff_handler(self):
+        top = Molecule.from_smiles("CC").to_topology()
+        parsley = ForceField("openff-1.0.0.offxml")
+
+        bogus_parameter_handler = ParameterHandler(version=0.3)
+        bogus_parameter_handler._TAGNAME = "bogus"
+        parsley.register_parameter_handler(bogus_parameter_handler)
+        with pytest.raises(
+            SMIRNOFFHandlersNotImplementedError, match="SMIRNOFF.*bogus"
+        ):
+            Interchange.from_smirnoff(force_field=parsley, topology=top)
+
+    def test_catch_bond_order_interpolation_bonds(self):
+        from openff.toolkit.tests.test_forcefield import xml_ff_bo
```

Comment:
I know this import is a bad pattern - I'd like to change how these XMLs are stored in the toolkit and then make those changes here.